_pages/developers.md

```diff
@@ -8,7 +8,7 @@ layout: single
 
 ## Language Specification
 
-CSL is defined by two documents: the human-readable [CSL specification](http://docs.citationstyles.org/en/1.0.1/specification.html), and the computer-readable [CSL schema](https://github.com/citation-style-language/schema/tree/v1.0.1), written in the [RELAX NG](http://relaxng.org/) compact syntax, which can be used for validation of CSL styles and locale files.
+CSL is defined by two documents: the human-readable [CSL specification](https://docs.citationstyles.org/en/stable/specification.html), and the computer-readable [CSL schema](https://github.com/citation-style-language/schema/tree/v1.0.2), written in the [RELAX NG](http://relaxng.org/) compact syntax, which can be used for validation of CSL styles and locale files.
```

Comment:
I hardcoded a 1.0.2 link here as I wasn't sure linking to the main branch would have been the best thing to do.